### PR TITLE
Update pagination.html

### PR DESCRIPTION
**Pull request by FinalAngel**
- add missing pagenav class
- use single quotes when nested

on load
image

moving to the next 5
image

moving to last
image

### CODE REVIEW

**Comment by vxsx** [comment]
:+1:

File: aldryn_newsblog/boilerplates/bootstrap3/templates/aldryn_newsblog/includes/pagination.html [hunk 1]
```diff
@@ -1,11 +1,18 @@
 {% load i18n %}
 
 {% if is_paginated %}
-    <ul>
+    <ul class="pagination">
         {% if page_obj.has_previous %}
+            {% if page_obj.number > 5 %}
```
Comment (vxsx) [inline aldryn_newsblog/boilerplates/bootstrap3/templates/aldryn_newsblog/includes/pagination.html]: can we not hardcode the number?

Comment (FinalAngel) [inline aldryn_newsblog/boilerplates/bootstrap3/templates/aldryn_newsblog/includes/pagination.html]: there is no setting yet /cc @mikek

**Comment by vxsx** [comment]
:+1:

**Comment by FinalAngel** [comment]
@czpython could you add here a Setting for the number limitations?

File: aldryn_newsblog/views.py [hunk 2]
```diff
@@ -188,6 +189,14 @@ class ArticleListBase(AppConfigMixin, AppHookCheckMixin, TemplatePrefixMixin,
     model = Article
     show_header = False
 
+    # Integer that indicates the number after which
+    # we'll start compacting the pagination pages into chunks.
+    pagination_pages_start = getattr(
+        settings, 'ALDRYN_NEWSBLOG_PAGINATION_PAGES_START', 10)
+    # The number of pages visible on each side (left and right).
+    pagination_pages_visible = getattr(
+        settings, 'ALDRYN_NEWSBLOG_PAGINATION_PAGES_VISIBLE', 4)
```
Comment (FinalAngel) [inline aldryn_newsblog/views.py]: @czpython we probably have to add these settings within aldryn as well or? Or can we include them in the namespace settings as we do for "pagination size"?

**Comment by mikek** [comment]
LGTM

**Pull request merged**